src/actions/posts.js

```diff
@@ -11,6 +11,7 @@ import {getCurrentChannelId, getMyChannelMember as getMyChannelMemberSelector} f
 import {getCustomEmojisByName as selectCustomEmojisByName} from 'selectors/entities/emojis';
 import {getConfig} from 'selectors/entities/general';
 import * as Selectors from 'selectors/entities/posts';
+import * as PostActions from 'actions/posts';
```

Comment:
Is this necessary?  It's importing the same file, right?

Reply:
good point, thanks!